doc/src/manual/ranch.set_protocol_options.asciidoc

```diff
@@ -34,9 +39,12 @@ The atom `ok` is always returned. It can be safely ignored.
 
 == Examples
 
-.Set the protocol options
+.Update the protocol options
```

Comment:
Sure but these are two separate things so should be shown as two separate examples.

Reply:
OK. I thought the single line example was a bit too trivial and since it doesn't show what the options consist of, I thought it was a good idea to extend it. But I'll keep both examples instead then.